scripts/prepare_ci.sh

```diff
@@ -79,9 +79,26 @@ download_files() {
     # Create directories if they don't exist
     mkdir -p "$target_dir" "$js_target_dir"
 
+    # Check if file exists
     # Download files to the specified directories
-    wget -P "$target_dir" "https://zkstuff.ams3.cdn.digitaloceanspaces.com/mopro/circom-examples-artifacts/${dir}/${circuit}_final.zkey"
-    wget -P "$js_target_dir" "https://zkstuff.ams3.cdn.digitaloceanspaces.com/mopro/circom-examples-artifacts/${dir}/${circuit}.wasm"
+    if ! [ -f "${target_dir}/${circuit}_final.arkzkey" ]; then
+        wget -P "$target_dir" "https://mopro.vivianjeng.xyz/${circuit}_final.arkzkey"
```

Comment:
I guess https://zkstuff.ams3.cdn.digitaloceanspaces.com/mopro/circom-examples-artifacts/keccak256/keccak256_256_test_final.arkzkey doesn't work cause arkzkey multiplier is missing? Can add it!

Reply:
oh is `arkzkey` available now?

Reply:
For all except multiplier2 because it is just a toy example, can generate and add it later if easier

Reply:
fixed